README-dest.md

```diff
@@ -0,0 +1,70 @@
+# My App
+
+This is the a web app based on BEdita web app.
+
+## Requirements
+
+1. PHP >= 7.2
+1. Download latest [Composer](https://getcomposer.org/doc/00-intro.md) or update via `composer self-update`.
+
+## Setup folders
+
+Check logs and tmp folder permissions.
+
+```bash
+sudo chown -R user:www tmp logs
+sudo chmod g+w -R user:www tmp logs
+```
```

Comment:
Before launch `composer install` `tmp` and `logs` don't exist. The install command takes cake of creation and permission setting on those folders.

Reply:
even permissions of web/apache group?

Reply:
Permissions are set to `0777` by default (see https://github.com/bedita/app/blob/3e2713ed757062d6ed83253eb799a763d510f409/src/Console/Installer.php#L136, `mkdir` is called with only the `path` parameter), which should grant anyone write permissions.

Reply:
The preferred way to setup permissions in a cakephp app is described here: https://book.cakephp.org/4/en/installation.html#permissions 

In local dev environment this works well for me, on production server we use other strategies and this is not needed

On manager app I prepared this script, we may create similar or same script here
https://github.com/bedita/manager/blob/master/bin/perms.sh